roborock/local_api.py

```diff
@@ -94,6 +94,13 @@ async def send_message(self, device_id: str, roborock_messages: list[RoborockMes
             return_exceptions=True,
         )
 
+    async def get_room_mapping(self, device_id: str) -> list[RoomMapping]:
```

Comment:
room_mapping in mqtt is unreliable from my experiences. Most of the time it gives no information back. Do you have different results?

Reply:
It looks reliable to me. Smoke testing it worked all times

Reply:
actually - it seems like it may work for mqtt now. Perhaps something has changed with the prefix changes? I know it used to not give any real results. I will move it

Reply:
Could be. Refactoring for local affected mqtt client, which fixed some issues with responses

Reply:
Okay fixed